src/olympia/lib/settings_base.py

```diff
@@ -1710,12 +1710,14 @@ def read_only_mode(env):
         'autocomplete-sort-param',
         'is-source-public-shim',
         'is-featured-addon-shim',
+        'disco-heading-and-description-shim',
     ),
     'v4': (
         'l10n_flat_input_output',
         'addons-search-_score-field',
         'ratings-can_reply',
         'ratings-score-filter',
+        'disco-heading-and-description-shim',
```

Comment:
I wasn't intending to have this gate for v4 but it's safest while we've still got the old discovery frontend deploy hanging around that consumes it.

Reply:
I think it's safe to remove that given that https://bugzilla.mozilla.org/show_bug.cgi?id=1673924 has been fixed on -dev/-stage.

Reply:
https://github.com/mozilla/addons-server/pull/15856/commits/914d7dd79e15c93c2089e6b0cb539489b0ccfe9e